include/torch-mlir/Dialect/Torch/Transforms/Passes.h

```diff
@@ -32,11 +32,20 @@ struct TorchLoweringPipelineOptions
   // If this option is false, only do the bare minimum for correctness.
   Option<bool> optimize{*this, "optimize", llvm::cl::desc("Do optimizations."),
                         llvm::cl::init(true)};
-  
+
   // If this option is false, decompose complex operations.
   // If this option is true, skip decomposition of complex operations.
-  Option<bool> decompose{*this, "decompose-complex-ops", llvm::cl::desc("Decompose complex operations."),
-                        llvm::cl::init(true)};                      
+  Option<bool> decompose{*this, "decompose-complex-ops",
+                         llvm::cl::desc("Decompose complex operations."),
+                         llvm::cl::init(true)};
+
+  // Disable decomposition rules for torch ops. Useless when
+  // decompose-complex-ops set to false.
+  Option<std::string> torchDecompositionSkipOps{
```

Comment:
This can be a `list<string>` I think. And it should not have `torch` in the name.